pysumma/__init__.py

```diff
@@ -8,6 +8,10 @@
 from .force_file_list import ForcingList
 from . import utils
 from .calibration import Ostrich, OstrichParam
+from . import hydroshare_utils
+from .preprocessing import LocalAttributes, InitialCondition, ParamTrial
```

Comment:
I think it might be good to have these not only live in `preprocessing` but be more general classes which act as interfaces to these methods. Then we could do something like `LocalAttributes.from_netcdf(filepath)` or `LocalAttributes.from_tif(filepath)` which might end up being a bit cleaner. Let's also make sure these follow the conventions in the `FileManager` class, so we should update `ParamTrial` -> `TrialParams`